### ISSUE TITLE
Pan Values Are Not Announced by Screen Reader

### ISSUE THREAD

**Issue by chigkim** [corrados/jamulus#498]
In audiomixerboard.cpp, the description is set `pPan->setAccessibleName ( tr ( "Local panning position of the current audio channel at the server" ) );`.
However, it never gets updated, so when keyboard focus lands on a pan control, screen reader just reads the above phrase even if you change the value with left/right arrows.
Need to fix CChannelFader::OnPanValueChanged to include `pPan->setAccessibleName ( QString::number(iPan) );`
This should trigger screen reader to announce new value when you change a pan with left/right arrows.
Also need to fix when initial pan value is set upon joining a server.
There are a number of blind Jamulus users who would love to see more accessibility improvement. Right now I'm just hacking things quickly and providing a custom build for them, so they could participate.
Theoretically, I could fork and send pull requests, but I'm blind myself, so I don't know if my hacks would alter the UI visually as well.
Mean time, I'll continue to open more issues and provide related snippets like this.
Happy to discuss on Jamulus if anyone is interested or has more ideas.

**Comment by corrados** [comment]
That is strange. I would have thought the Qt deals with such thing. Why is the application developer responsible for adding the QDial values to the accessible name. Maybe a Qt bug? Or something is wrong with the way Jamulus uses the Qt accessibility feature of Qt.

**Comment by chigkim** [comment]
Yes, it is strange because the volume control is working fine, but not pan.

**Comment by chigkim** [comment]
I finally got my windows environment to build. On windows, you can't even focus on the pan control with tab for some reason. It just skips it.

**Comment by corrados** [comment]
I just merged your pull request -> closed
